crates/namada/src/ledger/vp_host_fns.rs

```diff
@@ -41,6 +41,8 @@ pub enum RuntimeError {
     ReadPermanentValueError,
     #[error("Invalid transaction code hash")]
     InvalidCodeHash,
+    #[error("No value found in result buffer")]
+    NoValInResultBuffer,
```

Comment:
```suggestion
    NoValueInResultBuffer,
```
nit: IMO there's no reason to contract the identifier here, especially when `l` and `I` look pretty similar

Reply:
done in [5dda2da](https://github.com/anoma/namada/pull/2401/commits/5dda2daec6ecab1f55fc95c6d895368d430978c4)